### ISSUE TITLE
"oc get po" should use "oc get po -o wide" by default in order to show the host node of the pod

### ISSUE THREAD

**Issue by mrosic** [openshift/origin#5325]
"oc get po" shows 5 columns at the moment:
NAME                      READY     STATUS    RESTARTS   AGE

"oc get po -o wide" shows 6 columns: The additional column shows the host node of the pod.
NAME                      READY     STATUS    RESTARTS   AGE       NODE

I find the 6th column very useful. It would be nice if "oc get po" would by default show all 6 columns without me having to add "-o wide".

**Comment by deads2k** [comment]
I'm not sure I follow your workflow.  Why is it usually useful to know the node that a pod is running on?  I would expect the desire to be in the other direction: "what's running on this node?" or perhaps the question, "is this service spread across multiple nodes?".

I think that most end-users wouldn't be attached to particular nodes.

**Comment by mrosic** [comment]
It is useful for troubleshooting.

Say you are new to OpenShift and you created a router for the first time without a Virtual IP or any other fancy stuff. In that case it is vital to know on which node your router pod is running because without a Virtual IP your DNS entry is always resolving to a single node.

**Comment by deads2k** [comment]
> Say you are new to OpenShift and you created a router for the first time without a Virtual IP or any other fancy stuff. In that case it is vital to know on which node your router pod is running because without a Virtual IP your DNS entry is always resolving to a single node

So is your desire to know, "how can I access this pod?" without having a service, but using a `hostPort` on a container?  If so, would `oc status` be a good fit?  It would be able to introspect pods looking for pods using `hostPorts` and give you IP and port for them.

**Comment by mrosic** [comment]
No. In the example I gave it is important to know on which node your router is running because if's not running on the right node (that is the one your DNS entry is resolving to) your routes aren't going to work and you won't be able to access your applications via your web browser.

But that was just an example of the problems I faced while I was learning how to use OpenShift.

It's a minor change that I'm suggesting; one that helps people who are new to OpenShift understand what is going on in their cluster without typing/learning additional commands because "oc get po" is among the first things you will learn.

If you don't like the suggested change then I would open a new ticket and request that "oc get -h" properly documents that "-o wide" shows a 6th column with the name of the host node. That would be the second best thing to do. :)
